src/main/java/com/minecolonies/coremod/commands/generalcommands/ScanCommand.java

```diff
@@ -52,41 +51,47 @@ public ScanCommand(@NotNull final String... parents)
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final ActionMenu actionMenu) throws CommandException
     {
         // Will throw ClassCastException if null, but should never be null as these values are required.
+        final EntityPlayerMP player = actionMenu.getPlayerForArgument("player");
```

Comment:
Since player is optional, you want it above the comment that the other arguments are required.

Reply:
Yeah, good point